google_compute_engine/logger.py

```diff
@@ -32,6 +32,7 @@ def Logger(name, debug=False, facility=None):
   """
   logger = logging.getLogger(name)
   logger.handlers = []
+  logger.addHandler(logging.NullHandler())
```

Comment:
@illfelder does this library need to support 2.6? NullHandler isn't available pre-2.7.

See: https://github.com/google/google-api-python-client/blob/master/googleapiclient/__init__.py#L20